src/OrchardCore.Themes/TheAdmin/wwwroot/Styles/TheAdmin.css

```diff
@@ -9178,156 +9331,3 @@ select[multiple] {
     .persona .persona-content.small, .persona .persona-content.hint {
```

Comment:
Styles should not be added in the .css files directly.

Reply:
This is too old, I already schedule my time to revise the old PRs

Thanks again

Reply:
I agree. I kept the PR opened because there are things that can be reused.